<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qAndora</class>
 <widget class="QDialog" name="qAndora">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string> qAndora - Internet Radio</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/qAndora.png</normaloff>images/qAndora.png</iconset>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="albumLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Album</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="albumImage">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>261</width>
     <height>191</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>241</width>
     <height>191</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="stationBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>91</width>
     <height>261</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QToolButton" name="playPauseButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Pause</string>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/pause.png</normaloff>images/pause.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="skipButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Skip Song</string>
    </property>
    <property name="text">
     <string>Skip</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/skip.png</normaloff>images/skip.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="loveButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Mark Favorite</string>
    </property>
    <property name="text">
     <string>Love</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/favorite.png</normaloff>images/favorite.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="banButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Ban Song</string>
    </property>
    <property name="text">
     <string>Ban</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/ban.png</normaloff>images/ban.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="artistLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Artist</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
